spyder/locale/es/LC_MESSAGES/spyder.po

```diff
@@ -241,9 +240,8 @@ msgstr ""
 "para todas las sesiones)"
 
 #: spyder/app/mainwindow.py:810
-#, fuzzy
 msgid "Install Kite completion engine"
-msgstr "Usar un widget para completar texto"
+msgstr "Intalar motor de completado de Kite"
```

Comment:
*Instalar

Reply:
Thanks @andfoy. I'm reviewing @juanis2112 work right now and fixing errors like this one.